website/scripts/build_api_md.sh

```diff
@@ -1,32 +0,0 @@
-#!/bin/bash
```

Comment:
This file and all others in `website/scripts` should be kept. They are used for generating the markdown for the API from the source code. Can you restore them, please?

Reply:
fixed in 16ef333f72f5621e906d3ef6250bafffe756b72f